roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -39,9 +39,9 @@ netchecker_server_cpu_requests: 50m
 netchecker_server_memory_requests: 64M
 
 # Dashboard
-dashboard_enabled: false
+dashboard_enabled: true
 dashboard_image_repo: gcr.io/google_containers/kubernetes-dashboard-amd64
-dashboard_image_tag: v1.7.1
+dashboard_image_tag: v1.8.0
```

Comment:
Looks like v1.8.1 is out, can you change the tag value and also resync any yaml changes?